.github/workflows/make-pull-request/action.yml

```diff
@@ -0,0 +1,54 @@
+name: Making a pull request
+description: That action create a pull request with all changes
+inputs:
+  commit-path:
+    description: 'Path to the files that will be committed'
+    required: true
+  commit-message:
+    description: 'Message for commit'
+    required: true
+    default: 'Updating'
+  branch-name:
+    description: 'Name for creating branch'
+    required: true
+    default: 'update'
+  github-token:
+    description: 'Token for making operations with GitHub'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Commit files
+      id: commit
+      continue-on-error: true
+      run: |
+        git config --local user.email "41898282+github-actions[bot]@users.noreply.github.com"
+        git config --local user.name "github-actions[bot]"
+        git add ${{ inputs.commit-path }}
+        git commit -m "${{ inputs.commit-message }}"
+      shell: bash
+
+    - name: Push changes
+      if: steps.commit.outcome == 'success'
+      uses: ad-m/github-push-action@master
+      with:
+        github_token: ${{ inputs.github-token }}
+        branch: refs/heads/${{ inputs.branch-name }}-${{github.run_number}}
+
+    - uses: repo-sync/pull-request@v2
+      if: steps.commit.outcome == 'success'
+      name: pull-request
+      with:
+        source_branch: "${{ inputs.branch-name }}-${{github.run_number}}"
+        destination_branch: "master"
+        pr_title: ${{ inputs.commit-message }}
+        pr_body: |
+          That pull-request was generated automatically 🤖
+        pr_reviewer: |
```

Comment:
Looks like there is should be another format
https://github.com/repo-sync/pull-request#advanced-options

<img width="948" alt="Screenshot 2022-09-27 at 10 47 29" src="https://user-images.githubusercontent.com/40560660/192443077-80353bf4-3bdf-46d1-bbc7-27140b42b26d.png">